.github/workflows/render.yml

```
@@ -33,7 +37,7 @@ jobs:
       - name: 🚀 Deploy to GitHub pages
         # This allows CI to build branches for testing
         if: github.ref == 'refs/heads/main'
```

```suggestion
        if: (github.ref == 'refs/heads/main') && (matrix.python-version == '3.x')
```

Don't we need a check here so that this doesn't attempt to deploy to prod from _both_ matrix jobs and trigger a race?

(Perhaps ideally, we'd `include` a matrix value `deploy: true` to the `python-version: 3.x` matrix job and then key on that rather than explicit hardcoding `'3.x'` here, but that might be overcomplicating things.)

---

Yep, good idea!